ember-address-search/src/services/address-register.ts

```diff
@@ -18,21 +45,23 @@ export default class AddressRegisterService extends Service {
    * @param {string} fuzzyString The searched address, a fuzzy search will be performed
    * @returns {Array[Object]} The suggested addresses
    */
-  async suggest(fuzzyString) {
+  async suggest(fuzzyString: string): Promise<Address[] | null> {
```

Comment:
It seems we only return `null` if the service wasn't  configured correctly. It's a bit silly that we would have to check for null every time we use this method. It would probably be better to always assume it's a Promise and `assert` / `throw` instead of return null?